packages/devextreme/scss/widgets/base/splitter/_index.scss

```diff
@@ -11,16 +11,47 @@
   &.dx-splitter-vertical {
     flex-direction: column;
   }
+}
+
+.dx-resize-handle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
 
-  .dx-resize-handle {
-    flex-basis: 8px;
+  .dx-resize-handle-icon {
+    background-color: #424242;
```

Comment:
we shouldn't use certain colors in base styles

Reply:
since it's an icon now, this color style is removed